cmd/ore/aws/upload.go

```diff
@@ -236,6 +238,21 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		}
 	}
 
+	if len(uploadTags) > 0 {
+		hvmTagMap := make(map[string]string)
+		for _, tag := range uploadTags {
+			splitTag := strings.Split(tag, "=")
```

Comment:
Hmm, we should probably use `SplitN(tag, "=", 1)` here instead to handle values that have `=`?